tests/ui/transmutability/issue-101739-1.stderr

```diff
@@ -8,7 +8,7 @@ error: the constant `ASSUME_ALIGNMENT` is not of type `Assume`
   --> $DIR/issue-101739-1.rs:8:14
    |
 LL |         Dst: BikeshedIntrinsicFrom<Src, Context, ASSUME_ALIGNMENT>,
-   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `Assume`, found `bool`
```

Comment:
do we have the "machinery" in rustc to do the same thing as with exprs, for other paths so that this has a nicer span?

Reply:
No, this has to do with wfcheck which notoriously has bad spans, e.g. [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=3517b4458988344482c1b57c26c16e44). I've looked into it, doesn't seem to have a good solution.

Reply:
sad :(